avocado/plugins/run.py

```diff
@@ -162,9 +162,11 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
         job_run = job_instance.run()
-        result_dispatcher = ResultDispatcher()
-        if result_dispatcher.extensions:
-            result_dispatcher.map_method('render',
-                                         job_instance.result,
-                                         job_instance)
+
+        if job_instance.status != 'RUNNING':
```

Comment:
I think the output plugins should be always called. They receive the job_instance, therefor they can query for the status and treat it (eg. by storing the error or original status)